src/Common/src/Interop/Unix/libcrypto/Interop.ASN1.Print.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+using Internal.Cryptography;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class libcrypto
+    {
+        [DllImport(Libraries.LibCrypto)]
+        private static unsafe extern SafeAsn1StringHandle d2i_ASN1_type_bytes(IntPtr zero, byte** ppin, int len, Asn1StringTypeFlags flags);
+
+        [DllImport(Libraries.LibCrypto)]
+        private static extern int ASN1_STRING_print_ex(SafeBioHandle bio, SafeAsn1StringHandle str, Asn1StringPrintFlags flags);
+
+        internal static unsafe string DerStringToManagedString(
+            byte[] derString,
+            Asn1StringTypeFlags flags=AnyTextStringType)
+        {
+            SafeAsn1StringHandle asn1String = OpenSslD2I(
+                (zero, ppin, len) => d2i_ASN1_type_bytes(zero, ppin, len, flags),
```

Comment:
This is still going to create a closure and allocate a delegate due to ```flags```... is that intentional?  This might not be a critical path we need to worry about, so up to you.